tests/conftest.py

```diff
@@ -52,3 +54,14 @@ def minimal_openapi_spec_dict():
             }
         },
     }
+
+
+@pytest.fixture(scope='session', autouse=True)
+def mock_hashing():
```

Comment:
Can you double check if you can further reduce the scope of this session scoped autouse fixture? Maybe only use it on auth related tests and include them there. `autouse` sometimes can be a bit expensive-ish.

Reply:
Deleted fixture. Using this patch in necessary tests only.